.github/workflows/code_coverage.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       - name: Generate code coverage
         run: |
-          cargo +nightly tarpaulin --manifest-path packages/server/Cargo.toml --verbose --all-features --workspace --timeout 120 --out Xml
+          cargo +nightly tarpaulin --manifest-path packages/server/Cargo.toml --verbose --all-features --workspace --timeout 120 --ignore-tests --out Xml
```

Comment:
this means: "when running the code coverage tool, only check coverage for files that are not test files"
that's because covering test files is trivial (any code that appears in tests is automatically covered, because **it appears in tests**).